README.md

```diff
@@ -40,8 +40,8 @@ relationships are saved and can be viewed historically.
 
 1. Local Git repo and GitBash or similar shell-prompt installed/configured on your local machine.
 2. Set up GitHub by following the [instructions provided here](https://ikmdev.atlassian.net/wiki/spaces/IKM/pages/390201368/GitHub+Account+Creation+IKM+FDA+Shield)
-3. Download and install Open JDK Java 23 or greater [latest version](https://openjdk.org/projects/jdk/)
-4. Download and install Apache Maven 3.9 or greater [link here](https://maven.apache.org/download.cgi)
+3. Download and install Open JDK Java 24 or greater [latest version](https://openjdk.org/projects/jdk/)
```

Comment:
Can we change "where to get the JDK"? 

Azul's jdk download site: https://www.azul.com/downloads/?package=jdk#zulu
Azul's jdk download for 24 https://www.azul.com/downloads/?version=java-24&package=jdk#zulu